index.js

```diff
@@ -13,26 +13,26 @@ function Account (options) {
     throw new Error('options.url is required')
   }
 
+  var cacheKey = options.cacheKey || '_session'
   var state = {
-    cacheKey: options.cacheKey || '_session',
+    cacheKey: cacheKey,
     url: options.url,
-    validate: options.validate || function () {}
+    validate: options.validate || function () {},
+    session: getSession({cacheKey: cacheKey})
   }
 
-  getSession(state)
-
   return {
     get username () {
       return getUsername(state)
     },
-    signUp: require('./lib/signup').bind(this, state),
-    signIn: require('./lib/signin').bind(this, state),
-    signOut: require('./lib/signout').bind(this, state),
-    isSignedIn: require('./lib/is-signedin').bind(this, state),
-    get: require('./lib/get').bind(this, state, 'account'),
+    signUp: require('./lib/sign-up').bind(this, state),
+    signIn: require('./lib/sign-in').bind(this, state),
+    signOut: require('./lib/sign-out').bind(this, state),
+    isSignedIn: require('./lib/is-signed-in').bind(this, state),
+    get: require('./lib/get').bind(this, state),
     fetch: require('./lib/fetch').bind(this, state, 'account'),
     profile: {
-      get: require('./lib/get').bind(this, state, 'account.profile'),
```

Comment:
I added a `getProperties` helper method, so that we have a separate file for all our public APIs